test/mlir/driver/version.mlir

```diff
@@ -0,0 +1,4 @@
+// RUN: onnx-mlir --version | FileCheck %s
+// XFAIL: onnx-mlir-vendor
```

Comment:
The version is printed out (or *a* version anyway) regardless of whether vendor is specified. There should be a test for that.